Source/Components/Touchpoints/LiveImages/MLBusinessLiveImagesCarousel.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  MLBusinessLiveImagesCarouselView.swift
+//  MLBusinessComponents
+//
+//  Created by Lautaro Bonasora on 19/01/2023.
+//
+
+import Foundation
+
+protocol MLBusinessMultimediaCarousel: AnyObject {
+    func playOnFocus(currentCell: MLBusinessLiveImagesCellView)
+    func startByPosition(_ position: Int, cell: MLBusinessLiveImagesCellView, positionFocused: Int)
+    func restartFocused(_ position: Int, cell: MLBusinessLiveImagesCellView, positionFocused: Int)
+    func pauseFocused()
+    func updateConfig()
+}
+
+
+final class MLBusinessMultimediaCarouselImplementation: MLBusinessMultimediaCarousel {
+    private var lastCellFocused: MLBusinessLiveImagesCellView?
+    private var currentCellFocused: MLBusinessLiveImagesCellView?
+    
+    func startByPosition(_ position: Int, cell: MLBusinessLiveImagesCellView, positionFocused: Int) {
+        if IndexPath(indexes: [0,position]) == IndexPath(indexes: [0,0]) && IndexPath(indexes: [0,0]) == IndexPath(indexes: [0, positionFocused]) {
```

Comment:
Aca me perdi un poco. No sería lo mismo validar si `positon` y `positionFocused` son igual a 0? Onda, algo asi
```suggestion
        if position == 0 && 
        postionFocused == 0 {
```
O estamos haciendo la comparacion con un IndexPath por algo en particular?

Reply:
estas en lo cierto